notebooks/official/experiments/delete_outdated_tensorboard_experiments.ipynb

```diff
@@ -0,0 +1,667 @@
+{
```

Comment:
<p>has been -&gt; is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>